api/src/main/java/marquez/db/LineageDao.java

```diff
@@ -129,5 +129,14 @@ WHERE ds.uuid IN (<dsUuids>)""")
           + "    WHERE run_uuid=r.uuid\n"
           + "    GROUP BY run_uuid\n"
           + ") ro ON ro.run_uuid=r.uuid")
+  List<Run> getCurrentRunsWithFacets(@BindList Collection<UUID> jobUuid);
+
+  @SqlQuery(
+      "SELECT DISTINCT on(r.job_name, r.namespace_name) r.*, jv.version as job_version\n"
+          + "    FROM runs_view r\n"
+          + "    INNER JOIN job_versions jv ON jv.uuid=r.job_version_uuid\n"
+          + "    INNER JOIN jobs_view j ON j.uuid=jv.job_uuid\n"
+          + "    WHERE j.uuid in (<jobUuid>) OR j.symlink_target_uuid IN (<jobUuid>)\n"
+          + "    ORDER BY r.job_name, r.namespace_name, created_at DESC\n")
```

Comment:
I think we should use the more readable syntax variant as we update our queries.
```java
"""
SELECT DISTINCT on(r.job_name, r.namespace_name) r.*, jv.version as job_version
          FROM runs_view
          INNER JOIN job_versions jv ON jv.uuid=r.job_version_uuid
          INNER JOIN jobs_view j ON j.uuid=jv.job_uuid
          WHERE j.uuid in (<jobUuid>) OR j.symlink_target_uuid IN (<jobUuid>)
          ORDER BY r.job_name, r.namespace_name, created_at DESC
"""
```

Reply:
updated the syntax as you asked